src/elements/content-sidebar/activity-feed/activity-feed/ActivityFeed.js

```diff
@@ -179,6 +181,13 @@ class ActivityFeed extends React.Component<Props, State> {
         }
     };
 
+    handleToggleHeaderShadow = () => {
+        if (this.feedContainer) {
```

Comment:
I believe the scroll event should contain event target, along with the `scrollTop` information. Do we need to pull it from the ref if it's the same element?